packages/codemod/src/transforms/rename-js-to-jsx.ts

```diff
@@ -20,7 +20,7 @@ function renameJsToJsx(file: FileInfo, api: API, options: TRunnerOptions) {
     }
   }
 
-  return root.toSource(options);
+  return null;
```

Comment:
I'm curious about the nature of this change.
Was this causing a problem?

Reply:
I saw doing this (conditionally return `null` or the actual transformed content) in some of the example codemods. Essentially it's meant to return something only if something actually changed.

This transformer is actually kind of a fake one, because we're just renaming the file and we're not transforming anything. So for consistency I changed that to return `null` as it's more correct technically.

Reply:
Ok, so I guess the tool only updates the source file if the transformer returns something.

Thanks for the explanation 👍 .